src/wxProfilerGUI/profilergui.cpp

```diff
@@ -337,9 +337,12 @@ static std::vector<HANDLE> getThreadsByAttachMode(ProcessInfo& process_info)
 		return threadHandles;
 
 	case ATTACH_MOST_BUSY_THREAD:
-		if (HANDLE mostBusy = getMostBusyThread(process_info))
+	{
+		HANDLE mostBusy = getMostBusyThread(process_info);
+		if (mostBusy)
 			threadHandles.push_back(mostBusy);
 		return threadHandles;
+	}

```

Comment:
There might be a better way to silence this warning.

Depending on the compiler, sometimes wrapping the entire `if` expression in another set of parens does it.